### PR TITLE
XWIKI-13818:Apply same flamingo icone on blog (I hate git)

### PR DESCRIPTION
**Pull request by Pbast**
Previous commit (that I closed without purpose :-/). This one is correct

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogCode.xml [hunk 1]
```diff
@@ -835,15 +835,15 @@ $!xwiki.jsx.use($blogScriptsDocumentName)##
...
+(% class="plainmessage publish-message" %)((($services.icon.render('world') $message)))
 #end
 #**
  * Displays a message box with "show/hide" icon.
  * 
  * @param message A text message concerning blog article hiding
  *###
 #macro(hideMessageBox $message)
-(% class="plainmessage hide-message" %)((($message)))
+(% class="plainmessage hide-message" %)((($services.icon.render('lock_open') $message)))
```
Comment (gdelhumeau) [inline xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogCode.xml]: "lock_open" is actually the name of a "Silk" icon. You should use "unlock" instead. Actually, the allowed values are listed in the first column there: http://design.xwiki.org/xwiki/bin/view/Proposal/XWiki+Icon+Set

Thanks!

File: xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogStyle.xml [hunk 2]
```diff
@@ -286,11 +277,25 @@ div.edit-categories {
...
 .createPost {
-  background: transparent url("$xwiki.getSkinFile('icons/silk/add.png')") no-repeat scroll left center;
   font-size: 80%; 
   font-weight: bold;
-  padding-left: 18px;
-}</code>
+  padding-left: 9px;
+}
+div .publish-message P, div .hide-message P{
+  /* vertical align inside .publish-message and .hide-message */
```
Comment (gdelhumeau) [inline xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogStyle.xml]: Dead code should be removed.

File: xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/CategoriesCode.xml [hunk 3]
```diff
@@ -315,12 +315,12 @@
   #set($nameUrl = $escapetool.url($name))
   #foreach($i in [1..$level])*#end ##
 &lt;span class="blog-category-level"&gt;&lt;span class="blog-category"&gt;##
-&lt;a href="$services.rendering.escape($xwiki.getURL('Blog.CategoryRss', 'view', "xpage=plain&amp;amp;category=$nameUrl"), $doc.syntax)" title="RSS"&gt;&lt;img class="icon icon-manage" src="$xwiki.getSkinFile('icons/xwiki/rss-medium.png')" alt="[RSS]"/&gt;&lt;/a&gt;##
-&lt;span class="wikilink"&gt;&lt;a href="$services.rendering.escape($xwiki.getURL($name), $doc.syntax)"&gt;#getCategoryName($xwiki.getDocument($name)) &lt;span class="itemCount"&gt;($totalEntries)&lt;/span&gt;&lt;/a&gt;&lt;/span&gt;&lt;/span&gt; ##
+&lt;a href="$services.rendering.escape($xwiki.getURL('Blog.CategoryRss', 'view', "xpage=plain&amp;amp;category=$nameUrl"), $doc.syntax)" title="RSS"&gt;#toolImage('feed','[RSS]')&lt;/a&gt;&amp;nbsp;##
```
Comment (gdelhumeau) [inline xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/CategoriesCode.xml]: If we remove the `$alt` parameter of the macro, then you should change this call here.

File: xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogParameters.xml [hunk 4]
```diff
@@ -74,6 +74,7 @@
  * @param $imgName The name of the icon to use (without the extension).
  * @param alt The alternate text that should be displayed when images are not viewable.
```
Comment (Pbast) [inline xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogParameters.xml]: done!

**Comment by gdelhumeau** [comment]
Looks good. Thanks.

**Pull request merged**